lib/util.js

```diff
@@ -445,6 +445,11 @@ function formatValue(ctx, value, recurseTimes, ln) {
     }
   }
 
+  // Using an array here is actually better for the average case than using
+  // a Set. `seen` will only check for the depth and will never grow too large.
+  if (ctx.seen.indexOf(value) !== -1)
```

Comment:
`.includes()` unless it has a performance penalty?

Reply:
`indexOf` should be much faster than anything else.

Reply:
Maybe it is a case for @nodejs/v8 performance group to look into then.